package.json

```diff
@@ -5,14 +5,17 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./src/bin/www",
-    "devstart": "babel-watch ./src/index.js",
+    "devstart": "babel-watch --exec babel-node ./src/index.js",
     "build": "rm -rf dist && mkdir dist && babel src -s -d dist",
-    "test": " set NODE_ENV=test&& mocha -r @babel/register --recursive src/tests/ --exit",
+    "test": " set NODE_ENV=test && npm run reset:db && mocha -r @babel/register --recursive src/tests/ --exit --timeout 10000",
     "generate-lcov": "nyc report --reporter=text-lcov > lcov.info",
     "coveralls": "nyc report --reporter=text-lcov | coveralls",
     "codeclimate": "codeclimate-test-reporter > lcov.info",
     "coverage": "nyc npm test && npm run generate-lcov && npm run coveralls && npm run codeclimate",
-    "generate-report": "nyc --reporter=lcov --reporter=text-lcov npm test"
+    "generate-report": "nyc --reporter=lcov --reporter=text-lcov npm test",
+    "sequelize": "./node_modules/.bin/babel-node ./node_modules/.bin/sequelize $*",
+    "migrate": "npm run sequelize db:migrate",
+    "reset:db": "npm run sequelize db:migrate:undo:all && npm run migrate"
```

Comment:
Also for reset:db script `sequelize db:migrate:undo:all && npm run migrate"` should do